### PR TITLE
Finished task - http://www.google-melange.com/gci/task/view/google/gci2014/5617184554352640

### PR DESCRIPTION
**Pull request by tradzik**
I have finished this task for Google Code-In : http://www.google-melange.com/gci/task/view/google/gci2014/5617184554352640

Unfortunately, I can't claim it, because my "current" task is still pending review -.-
Please do not assign it to anybody other than me - this task is finished.

### CODE REVIEW

**Comment by roonyh** [comment]
Good job. Thank you! Can you be a little more helpful for the new students by adding more instructions on how to add their pictures and links? You can specify git commands, where they should copy there pictures and specific html tags they should put into the index.html file.

**Comment by tradzik** [comment]
Sure, I can and I will do that in the second (or tomorrow - it is midnight
in my timezone). Can please review my task?

Sun Dec 07 2014 at 23:23:14 użytkownik Aruna Herath <
notifications@github.com> napisał:

**Comment by roonyh** [comment]
Great!
I am sorry but I don't know much about Raspberry Pi. I'll ping the mentors about it.

**Comment by tradzik** [comment]
Thank you. I hope my task'll be reviewed quickly, because I want to finish
much more :-)

Sun Dec 07 2014 at 23:42:36 użytkownik Aruna Herath <
notifications@github.com> napisał:

**Comment by tradzik** [comment]
I have already did that what you asked for.

**Comment by roonyh** [comment]
Nice!!
Please use [inline html](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#inline-html) to add specific html changes that the students should make to add there picture to the site.

**Comment by roonyh** [comment]
And please submit a link to this pull request as work on your task.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+
+1) Sign up or login into GitHub (http://github.com/)
+
+2) Fork this (https://github.com/fossasia/fossasia.github.io) repository using button in right top corner.
+
+3) Enter in console `git clone https://github.com/YOURNICK/fossasia.github.io.git`
+
+4) Go to the folder using `cd fossasia.github.io` command
+
+5) Copy your photo to `images/students` directory rescaling it before (to size about 300px * 270 px)
```
Comment (roonyh) [inline README.md]: It should be square. so lets say 300px*300px

Comment (tradzik) [inline README.md]: Thank you for tips. I will correct that when I will come back from school

**Comment by roonyh** [comment]
This is a great opportunity for you to learn about [git squashing](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html). It will be a nice challenge and would come handy for you in future. Its fine if you are busy with a task and not want to get into this right now. But want to give it a try and squash last 6 commits into one?

**Comment by tradzik** [comment]
Ok, I'll read about that. Currently I am working on task (you're mentoring
:-) ) and will take care of that later

**Pull request merged**